spec/features/student/student_can_create_a_clone_spec.rb

```diff
@@ -26,4 +26,35 @@
 
     expect(current_path).to eq(root_path)
   end
+
+  scenario 'admin can delete clone' do
```

Comment:
Could we move this to another file? Maybe a `spec/features/admin/admin_can_delete_a_clone_spec.rb`?